src/audio/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ set(eq-fir_sources eq_fir/eq_fir.c eq_fir/eq_fir_generic.c)
 set(eq-iir_sources eq_iir/eq_iir.c eq_iir/iir.c)
 set(dcblock_sources dcblock/dcblock.c dcblock/dcblock_generic.c)
 set(crossover_sources crossover/crossover.c crossover/crossover_generic.c)
-set(tdfb_sources tdfb/tdfb.c tdfb/tdfb_generic.c)
+set(tdfb_sources tdfb/tdfb.c tdfb/tdfb_generic.c tdfb/tdfb_direction.c eq_iir/iir.c tdfb/tdfb_math.c)
```

Comment:
I need to move more IIR functions to library. Also @ShriramShastry 's work copied here to math will be in lib.

Reply:
Done in this version